std/traits.d

```diff
@@ -1087,6 +1079,38 @@ template ParameterStorageClassTuple(func...)
     static assert(pstc[2] == STC.none);
 }
 
+/*****************
+ * Convert string tuple Attribs to ParameterStorageClass bits
+ * Params:
+ *      Attribs = string tuple
+ * Returns:
+ *      ParameterStorageClass bits
+ */
+ParameterStorageClass extractParameterStorageClassFlags(Attribs...)()
```

Comment:
Done. Interesting trick to avoid generating code into the object file for functions only executed at compile time. This should also avoid the coverage tests complaining.